src/modules/ui/controls/play/play.ts

```diff
@@ -81,18 +75,11 @@ export default class PlayControl implements IPlayControl {
     this._togglePlayback = this._togglePlayback.bind(this);
   }
 
-  private _bindEvents() {
```

Comment:
Do you plan to add another subscription to playback state? Because without subscription control view will not be updated

Reply:
Nice catch. I returned this part and it seems to work well.